tests/unit/lms/services/application_instance_getter_test.py

```diff
@@ -12,18 +12,19 @@
 
 class TestApplicationInstanceGetter:
     def test_developer_key_returns_the_developer_key(self, ai_getter):
-        assert ai_getter.developer_key("TEST_CONSUMER_KEY") == "TEST_DEVELOPER_KEY"
+        assert ai_getter.developer_key() == "TEST_DEVELOPER_KEY"
 
     def test_developer_key_returns_None_if_ApplicationInstance_has_no_developer_key(
         self, ai_getter, test_application_instance
     ):
         test_application_instance.developer_key = None
 
-        assert ai_getter.developer_key("TEST_CONSUMER_KEY") is None
+        assert ai_getter.developer_key() is None
 
+    @pytest.mark.usefixtures("unknown_consumer_key")
```

Comment:
I like this pattern. Prevents the "we don't use this fixture, did we import it by mistake?" question